submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/exception/SerializationExceptionHandler.kt

```diff
@@ -0,0 +1,41 @@
+package ac.uk.ebi.biostd.submission.web.exception
+
+import ac.uk.ebi.biostd.submission.domain.exception.InvalidExtensionException
+import ac.uk.ebi.biostd.tsv.deserialization.model.TsvChunk
+import ac.uk.ebi.biostd.validation.SerializationException
+import ebi.ac.uk.errors.ValidationNode
+import ebi.ac.uk.errors.ValidationNodeStatus.ERROR
+import ebi.ac.uk.errors.ValidationTree
+import ebi.ac.uk.errors.ValidationTreeStatus.FAIL
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.ResponseStatus
+
+@ControllerAdvice
+class SerializationExceptionHandler {
+    @ResponseBody
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(value = [SerializationException::class])
+    fun handle(exception: SerializationException): ValidationTree {
+        val node = ValidationNode(ERROR, "Error processing submission", getErrors(exception))
+        return ValidationTree(FAIL, node)
+    }
+
+    fun getErrors(exception: SerializationException): List<ValidationNode> =
```

Comment:
This is just an aesthetic point but, could you please move these helper functions after the exception handlers? I think it looks weird to have them there in the middle 😅